docs/cloud/alerts-notifications/add-webhook-notification-configuration.md

```diff
@@ -0,0 +1,105 @@
+
+#### Webhook service
+
+A webhook integration allows your application to receive real-time alerts from Netdata by sending HTTP requests to a specified URL. In this document, we'll go over the steps to set up a generic webhook integration, including adding headers, and implementing different types of authorization mechanisms.
+
+##### Netdata webhook integration
+
+A webhook integration is a way for one service to notify another service about events that occur within it. This is done by sending an HTTP POST request to a specified URL (known as the "webhook URL") when an event occurs.
+
+Netdata webhook integration service will send alert notifications to the destination service as soon as they are detected.
+
+The notification content sent to the destination service will be a JSON object having these properties:
+
+| field   | type   | description |
+| :--     | :--    | :--         |
+| message | string | A summary message of the alert. |
+| alarm | string | The alarm the notification is about. |
+| info | string | Additional info related with the alert. |
+| chart | string | The chart associated with the alert. |
+| context | string | The chart context. |
+| space | string | The space where the node that raised the alert is assigned. |
+| family | string | Context family. |
+| class | string | Classification of the alert, e.g. "Error". |
+| severity | string | Alert severity, can be one of "warning", "critical" or "clear". |
+| date | string | Date of the alert in ISO8601 format. |
+| duration | string |  Duration the alert has been raised. |
+| critical_count | integer | umber of critical alerts currently existing on the same node. |
+| warning_count | integer | Number of warning alerts currently existing on the same node. |
+| alarm_url | string | Netdata Cloud URL for this alarm. |
+
+##### Extra headers
+
+When setting up a webhook integration, the user can specify a set of headers to be included in the HTTP requests sent to the webhook URL.
+
+By default, the following headers will be sent in the HTTP request
+
+|            **Header**            | **Value**                 |
+|:-------------------------------:|-----------------------------|
+|     Content-Type             | application/json        |
+
+##### Authorization mechanism
+
+Netdata webhook integration supports 3 different authorization mechanisms:
+
+1. Mutual TLS (recommended)
+
+In mutual Transport Layer Security (mTLS) authorization, the client and the server authenticate each other using X.509 certificates. This ensures that the client is connecting to the intended server, and that the server is only accepting connections from authorized clients.
+
+To take advantage of mutual TLS, you can configure your server to verify Netdata's client certificate. To do that you need to download our [CA certificate file](http://localhost) and configure your server to use it as the
```

Comment:
@hugovalente-pm  CA certificate file, is the link correct I think something is missing

Reply:
ping @hugovalente-pm